extensions/storage-system/pkg/config/config.go

```diff
@@ -76,7 +74,7 @@ type Drivers struct {
 
 type OCISDriver struct {
 	// Root is the absolute path to the location of the data
-	Root string `yaml:"root" env:"STORAGE_SYSTEM_OCIS_ROOT"`
+	Root string `yaml:"root" env:"STORAGE_SYSTEM_OCIS_ROOT" desc:"Path for the persistence directory."`
```

Comment:
To be used for what?

Reply:
"persistence directory" = the storage-system service does store it's data here. I though that should be clear from the description. 

Do you have a better wording?

Reply:
As developer you are most likely knowledged, but keep in mind that readers have not the knowledge you have...
Suggestion:
`Path for the directory where the storage-system service stores it's persistent data.`

Question, is there a default already defined? and if not what would be a suggestion?

Reply:
Defaults are not changed in comparison to the current state on owncloud.dev, see:
![image](https://user-images.githubusercontent.com/34452982/176106826-ad4353ce-d216-422d-96cf-64d2fab10f3d.png)